packages/block-library/src/image/image.js

```diff
@@ -694,7 +694,10 @@ export default function Image( {
 					setAttributes( {
 						width: `${ elt.offsetWidth }px`,
 						height: 'auto',
-						aspectRatio: `${ ratio }`,
+						aspectRatio:
+							ratio === naturalRatio
+								? undefined
```

Comment:
Does this explicitly need to be `'auto'`? In most cases undefined/unset is treated as auto.

Reply:
The difference is that setting `'auto'` will output the CSS whereas `undefined` will output nothing.

Reply:
Looks like you changed it already :)